x-pack/plugin/downsample/src/internalClusterTest/java/org/elasticsearch/xpack/downsample/ILMDownsampleDisruptionIT.java

```diff
@@ -172,6 +173,14 @@ public void testILMDownsampleRollingRestart() throws Exception {
         startDownsampleTaskViaIlm(sourceIndex, targetIndex);
         assertBusy(() -> assertTargetIndex(cluster, targetIndex, indexedDocs));
         ensureGreen(targetIndex);
+        // We assert that ILM successfully completed the phase
+        logger.info("Waiting for ILM to complete the phase for index [{}]", targetIndex);
+        ClusterServiceUtils.addTemporaryStateListener(clusterState -> {
+            IndexMetadata indexMetadata = getIndexMetadataByIndexName(clusterState, targetIndex);
```

Comment:
Integration tests don't need to be project-aware yet, as we don't have any way of making them work with different projects.


```suggestion
            IndexMetadata indexMetadata = clusterState.metadata().getProject().index(targetIndex);
```